src/core/style/mixins.scss

```diff
@@ -251,4 +251,21 @@
     cursor: default;
   }
 
+  &.md-indeterminant ._md-icon{
+    &:after {
+        box-sizing: border-box;
```

Comment:
Too much indent. You're IDE is likely configured to the wrong indent for `.scss` files.